client/src/apis/user.ts

```diff
@@ -0,0 +1,11 @@
+import { baseRequest } from './util';
+
+export const getWorkspaces = async (userId: number) => {
```

Comment:
api 타입 같은 경우 apiTypes 디렉토리를 types에 하나 더 만들어서 `d.ts`파일로 관리하는거 어떨까요?
req랑 res 두개로 관리하는거에요
```suggestion
declare namespace req {
  export type Workspace = {
    userId: number;
  };
  // etc
}

declare namespace res {
  export type Workspace = {
    message: 타입어쩌고;
    data: 타입어쩌고;
  };
  // etc
};
```

- 사용할 때
```ts
export const getWorkspaces = async ({ userId } : req.Workspace): Promise<res.Workspace> => {
    const { data } = await baseRequest.get(`/user/${userId}/workspace`);
    return data;
};
```

Reply:
오 이렇게 하면 관리하기 편할것 같아요

Reply:
타입관련된건 다른 이슈 만들어서 처리해볼게요!